src/main/java/com/github/splendor_mobile_game/game/model/TokenList.java

```diff
@@ -8,17 +8,71 @@ public class TokenList {
 
     private ArrayList<Token> tokens;
 
-    private TokenType tokenType;
+    private final TokenType tokensType;
 
-    public TokenList(TokenType tokenType) {
-        this.tokenType = tokenType;
+    private int availableTokens = 7;
+
+    public TokenList(TokenType tokensType) {
+        this.tokensType = tokensType;
     }
 
-    public boolean addToken(Token token) {
-        if (tokenType != token.getTokenType()) return false;
+    public void addToken(Token token) {
+        if (tokensType != token.getTokenType()) return;
 
         this.tokens.add(token);
-        return true;
     }
 
+
+
+
+    public int getAvailableTokensCount() {
+        return availableTokens;
+    }
+
+
+    public int getEquippedTokensCount() {
+        return 7 - availableTokens;
```

Comment:
The number `7` there will vary depending on the numbers of players. Please, add a TODO comment there and later we will add some issue about that.